External/FEXCore/include/FEXCore/Utils/DeferredSignalMutex.h

```diff
@@ -11,6 +11,84 @@
 #include <unistd.h>
 
 namespace FEXCore {
+#ifndef _WIN32
+  class ForkableUniqueMutex final {
```

Comment:
Needs a docstring explaining why `std::*mutex`es can't be forked reliably.

Reply:
Added a docstring.

Reply:
Changed the comment over to the quoted one.